test-network-function/certification/suite.go

```diff
@@ -68,9 +69,56 @@ var _ = ginkgo.Describe(common.AffiliatedCertTestKey, func() {
 
 		testContainerCertificationStatus()
 		testAllOperatorCertified(env)
+		testHelmCertified(env)
 	}
 })
 
+func testHelmCertified(env *configpkg.TestEnvironment) {
+	testID := identifiers.XformToGinkgoItIdentifier(identifiers.TestHelmIsCertifiedIdentifier)
+	ginkgo.It(testID, ginkgo.Label(testID), func() {
+		certAPIClient = api.NewHTTPClient()
+		helmcharts := env.HelmchartsUnderTest
+		if len(helmcharts) == 0 {
+			ginkgo.Skip("No helm charts to check")
+		}
+		out, _ := certAPIClient.GetYamlFile()
+		if out.Entries == nil {
+			ginkgo.Skip("No helm charts from the api")
+		}
+		ourKubeVersion := GetKubeVersion()[1:]
+		failedHelms := []configsections.HelmChart{}
+		for _, helm := range helmcharts {
+			certified := false
+			for _, v := range out.Entries {
+				for _, val := range v {
+					if val.Name == helm.Name && val.Version == helm.Version {
+						if val.KubeVersion != "" {
+							if CompareVersion(ourKubeVersion, val.KubeVersion) {
+								certified = true
+								break
+							}
+						} else {
+							certified = true
+							break
+						}
+					}
+				}
+				if certified {
+					log.Info(fmt.Sprintf("Helm %s with version %s is certified", helm.Name, helm.Version))
+					break
+				}
+			}
+			if !certified {
+				failedHelms = append(failedHelms, helm)
+			}
+		}
+		if len(failedHelms) > 0 {
+			log.Warnf("Helms that are not certified: %+v", failedHelms)
```

Comment:
this is not a warning, it's an error
``log.Error("Helms that are not certified: %+v", failedHelms)``
Also, please print it in the claim file as well
``tnf.ClaimFilePrint("Helms that are not certified: %+v", failedHelms)``